ios/RCTGoogleAnalyticsBridge/RCTGoogleAnalyticsBridge/RCTGoogleAnalyticsBridge.m

```diff
@@ -377,6 +377,13 @@ - (NSDictionary *)constantsToExport
     [tracker send:[builder build]];
 }
 
+RCT_EXPORT_METHOD(dispatch:(RCTPromiseResolveBlock)resolve
+                  reject:(__unused RCTPromiseRejectBlock)reject) {
+    [[GAI sharedInstance] dispatchWithCompletionHandler:^void(GAIDispatchResult result){
+        resolve(result != kGAIDispatchError ? @YES : @NO);
```

Comment:
Do a `reject` if it has an error here instead of resolving `@NO`.

E.g.
```obj-c
if (result != kGAIDispatchError) {
    resolve(@YES);
} else {
    reject(@"DISPATCH_FAILED", nil, RCTErrorWithMessage(@"Dispatch failed"));
}
```

Remove `___unused` on the rejection-block parameter type as well then. :)